### PR TITLE
Feature/dsi 2942

### PR DESCRIPTION
**Pull request by Univer-lion-lab**

### CODE REVIEW

File: Backend/backend.template.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "parameters": {
         "templatesBaseUri": {
             "type": "string",
-            "defaultValue": "https://raw.githubusercontent.com/DFE-Digital/login.dfe.infrastructure/master/"
+            "defaultValue": "https://raw.githubusercontent.com/DFE-Digital/login.dfe.infrastructure/feature/DSI-3165-Set-HTTP-version-2.0-Interactions/"
```
Comment (ageerlenin) [inline Backend/backend.template.json:7]: Please remove reference to feature branches when merging into master

**Review by ageerlenin — CHANGES_REQUESTED** [review]

File: Backend/backend.template.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "parameters": {
         "templatesBaseUri": {
             "type": "string",
-            "defaultValue": "https://raw.githubusercontent.com/DFE-Digital/login.dfe.infrastructure/master/"
```
Comment (ageerlenin) [inline Backend/backend.template.json:7]: please correct typo

File: Frontend/frontend.template.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "parameters": {
         "templatesBaseUri": {
             "type": "string",
-            "defaultValue": "https://raw.githubusercontent.com/DFE-Digital/login.dfe.infrastructure/master/"
+            "defaultValue": "https://raw.githubusercontent.com/DFE-Digital/login.dfe.infrastructure/master"
```
Comment (ageerlenin) [inline Frontend/frontend.template.json]: please correct typo

File: Shared/private-endpoint.template.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "parameters": {
     "templatesBaseUri": {
       "type": "string",
-      "defaultValue": "https://raw.githubusercontent.com/DFE-Digital/login.dfe.infrastructure/master/"
+      "defaultValue": "https://raw.githubusercontent.com/DFE-Digital/login.dfe.infrastructure/master"
```
Comment (ageerlenin) [inline Shared/private-endpoint.template.json]: please correct typo

File: Shared/redis-cache-private-endpoint.json [hunk 5]
```diff
@@ -4,7 +4,7 @@
   "parameters": {
     "templatesBaseUri": {
       "type": "string",
-      "defaultValue": "https://raw.githubusercontent.com/DFE-Digital/login.dfe.infrastructure/master/"
+      "defaultValue": "https://raw.githubusercontent.com/DFE-Digital/login.dfe.infrastructure/master"
```
Comment (ageerlenin) [inline Shared/redis-cache-private-endpoint.json]: please correct typo

**Review by ageerlenin — CHANGES_REQUESTED** [review]

**Review by Univer-lion-lab — COMMENTED** [review]
All feature branches have removes

**Review by ageerlenin — APPROVED** [review]

**Pull request merged**